augur/tasks/github/facade_github/tasks.py

```diff
@@ -262,7 +264,20 @@ def insert_facade_contributors(self, repo_git):
 
     key_auth = GithubRandomKeyAuth(logger)
 
-    process_commit_metadata(logger, key_auth, list(new_contribs), repo_id, platform_id)
+    # Process results in batches to reduce memory usage
+    batch = []
+    BATCH_SIZE = 1000
```

Comment:
`BATCH_SIZE` should ideally be a configurable parameter instead of hardcoded here.